gslb/ingestion/gslb.go

```diff
@@ -732,7 +735,8 @@ func Initialize() {
 	// check whether we already have a GSLBConfig object created which was previously accepted
 	// this is to make sure that after a reboot, we don't pick a different GSLBConfig object which
 	// wasn't accepted.
-	CheckGSLBConfigsAndInitialize()
+	// TODO
+	alreadyConfigured := CheckGSLBConfigsAndInitialize()
```

Comment:
It seems this variable is not getting used anywhere. Did you mean to check whether `alreadyConfigured` is true/false? This is also causing the build job to fail.